### ISSUE TITLE
Query

### ISSUE THREAD

**Issue by malikfahadsarwar** [sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning#203]
can you explain that do we use the teacher forcing in the validation phase EVEN  with no parmater update  in order to compute the cross entropy loss like we do in training to measure model performance in terms of overfitting,underfitting,generalization  not used the autoregressive technique in the validation set? autoregressive is ONLY used in test phase not in validation?
THANKS
